src/utils/functions/render-program-section/renderProgramSection.test.tsx

```
@@ -41,12 +88,18 @@ describe('renderProgramSection', () => {
             { id: 4, url: 'img4', mimeType: 'image/jpeg' },
         ],
     };
+
     const baseHandlers = {
         onTitleChange: jest.fn(),
         onDescriptionChange: jest.fn(),
         onImagesChange: jest.fn(),
     };
 
+    beforeEach(() => {
+        Object.keys(capturedProps).forEach((k) => delete capturedProps[k]);
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)